imap-proto/src/parser/rfc3501.rs

```diff
@@ -13,6 +13,7 @@ use nom::IResult;
 use std::str;
 
 use crate::parser::rfc4551;
+use crate::parser::rfc5464::*;
```

Comment:
Okay, please don't use `*` imports -- sorry, should have mentioned that the first time around.